lives-of-primitive-quakers/modernized/18-chapter-17.adoc

```diff
@@ -420,7 +426,7 @@ prevail upon the civility of your nature; for they will oppress the innocent lif
 and bring grievous weights and burdens upon the soul.
 . . Oh beware of this compliance!
 Let me put you in mind of that sensible resolution, so frequently,
-and so passionately repeated by you: '`Il faut queje rompe!--il faut que je rompe.`'
+and so passionately repeated by you: '`Il faut queje rompe!--il faut que je rompe.`' [I must be broken. I must be broken]
```

Comment:
does this render right? do we need to escape the bracket here? `+++[+++I must be`...?